src/sentry/static/sentry/app/components/group/pluginActions.jsx

```diff
@@ -126,38 +119,22 @@ const PluginActions = createReactClass({
             style={{borderBottom: '1px solid rgb(221, 221, 221)'}}
           >
             <li className={actionType == 'create' ? 'active' : ''}>
-              <a id="create" onClick={this.handleClick}>
-                Create
-              </a>
+              <a onClick={() => this.handleClick('create')}>{t('Create')}</a>
             </li>
             <li className={actionType == 'link' ? 'active' : ''}>
-              <a id="link" onClick={this.handleClick}>
-                Link
-              </a>
+              <a onClick={() => this.handleClick('link')}>{t('Link')}</a>
             </li>
           </ul>
-          {this.state.showModal && actionType == 'create' &&
-            !this.state.pluginLoading && (
-              <Modal.Body>
-                {plugins.get(plugin).renderGroupActions({
-                  plugin,
-                  group: this.getGroup(),
-                  project: this.getProject(),
-                  organization: this.getOrganization(),
-                  actionType: 'create',
-                  onSuccess: this.closeModal,
-                })}
-              </Modal.Body>
-            )}
-          {this.state.showModal && actionType == 'link' &&
+          {this.state.showModal &&
+            actionType &&
             !this.state.pluginLoading && (
               <Modal.Body>
                 {plugins.get(plugin).renderGroupActions({
                   plugin,
                   group: this.getGroup(),
                   project: this.getProject(),
                   organization: this.getOrganization(),
-                  actionType: 'link',
+                  actionType,
```

Comment:
Does this not have the same issue as externalIssueActions (because `renderGroupActions` is not an `AsyncComponent`?)

Reply:
oh I put the key in the Modal not the Modal.Body - I can move that down